### DISCUSSION TITLE
How to check which snippets are loaded?

### DISCUSSION THREAD

**Discussion by akim-13** [L3MON4D3/LuaSnip#767] [Q&A] [answered]
I just noticed that when I type in markdown files `u<TAB>` a `<>` is inserted, or `l<TAB>` inserts `[]()`. This is quite annoying and I don't know how to track down where it's coming from. Is there any way to check what might be causing this? My configuration can be found [here](https://github.com/akim-13/neovim/tree/dev/lua/rc) if it helps.

**Comment by L3MON4D3** [comment, marked as answer]
Mhhh, you can try `require("luasnip.extras.snippet_list").open()`, that should list all snippets, and which filetype they came from

**Reply by L3MON4D3** [reply]
Or shorter, `:lua =ls.available()`

**Reply by akim-13** [reply]
How do I run it form vim cmd?

**Reply by leiserfg** [reply]
Type exactly that in normal mode.
Edit: well you need ls so better:
`:lua= require("luasnip").available()`
